source/core/src/main/com/csse3200/game/components/minigame/MiniGamesScores.java

```diff
@@ -0,0 +1,147 @@
+package com.csse3200.game.components.minigame;
+
```

Comment:
64% coverage

Reply:
Yep, will implement more tests as the code is completed for that with all 3 minigames.